packages/lodestar/test/unit/network/reqresp/encoders/request.test.ts

```diff
@@ -0,0 +1,73 @@
+import chai, {expect} from "chai";
```

Comment:
Merged requestEncode and requestDecode tests in a single file with common test cases